src/h5web/App.test.tsx

```diff
@@ -21,48 +21,52 @@ function renderApp(): RenderResult {
 }
 
 test('renders with root group selected', async () => {
-  const { findByText, getByRole } = renderApp();
+  renderApp();
 
-  const noVisText = await findByText('Nothing to visualize');
+  const noVisText = await screen.findByText('Nothing to visualize');
```

Comment:
`screen` makes tests easier to write thanks to auto-completion. Adding each method to the destructuring assignment takes a lot longer.